docs/integrations/databases/opentelemetry/mongodb-opentelemetry.md

```diff
@@ -257,24 +277,54 @@ Use this dashboard to:
 
 <img src='https://sumologic-app-data-v2.s3.amazonaws.com/dashboards/MongoDB-OpenTelemetry/MongoDB-Query-Logs.png' alt="Query Logs" />
 
+### Sharding
+
+The **MongoDB - Sharding** dashboard shows sharding related errors, events, failures and number of chunks moving between shards.
+
+Use this dashboard to:
+
+-   Identify Sharding errors and warnings.
+-   Gain insights into Chunk operations.
+
+<img src='https://sumologic-app-data-v2.s3.amazonaws.com/dashboards/MongoDB-OpenTelemetry/MongoDB-Sharding.png' alt="Sharding" />
+
 ### Replication Logs
 
-The **MongoDB - Replication Logs** dashboard shows replica deletes/updates/inserts trend and replica state.
+The **MongoDB - Replication Logs** dashboard shows replication events, errors, warnings, and nodes.
```

Comment:
Have we made changes to this dashboard . This seem to be a log dashbaord.. and we have added replication related metric dashboard seperately. Why are we changing the description for this dashboard ?

Reply:
fixed it.